ark/utils/notebooks_test.py

```diff
@@ -106,3 +115,36 @@ def test_segment_image_data_folder(tb):
 
         # create the expression matrix
         notebooks_test_utils.create_exp_mat(tb)
+
+
+# test mibitiff clustering
+@testbook(FLOWSOM_CLUSTER_PATH, timeout=6000)
+def test_flowsom_cluster_mibitiff(tb):
```

Comment:
Each of these tests takes about 30 seconds. Once we figure out the Travis R installation issue, this will become the next biggest time sync. We can probably use images that are like 50x50 pixels, instead of 1024x1024, to dramatically speed this up.

Reply:
After removing all but 7 TIFs (the ones Candace said would pop up most frequently), I think mocking up the call to `cluster_pixels` (which implicitly calls `som_runner.R`) will provide the next biggest speed boost. Let's see how much of a speedup that provides, if it's still too much then I can reduce the size of the images.